src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -112,7 +112,6 @@ const ProfileHeader = ({
               alt={`${latestBranchOfService} seal`}
               className="profile-service-badge vads-u-padding-right--3"
               aria-hidden="true"
-              role="presentation"
```

Comment:
@jenstrickland For visibility: Axe did not like this `role` attribute on the image. Changing it to `role="img"` made the warning go away, but after cross checking with other parts of our application, it appears we don't usually add the `role` attribute to images anyway, so I removed it.

Reply:
@sanlouise Would you please add the `role="img"` back in. We do add a role for some images, such as decorative images like this. Presentation or img is fine. More reading: http://john.foliot.ca/aria-hidden/

Reply:
@sanlouise && @jenstrickland I have a feeling axe threw a warning because this role was added to an `<img />` tag with alt text. In that case, the implicit role would be "image" and because we have alt text, that's a marker that the image is important and should be announced to the screen reader.

You might try updating to the following to ignore the image for assistive tech:

```diff
<img
  src={SERVICE_BADGE_IMAGE_PATHS.get(latestBranchOfService)}
- alt={`${latestBranchOfService} seal`}
+ alt=""
  className="profile-service-badge vads-u-padding-right--3"
- aria-hidden="true"
/>
```

Reply:
Ooooh, great catch, @1Copenut ! 

If this image has alt text, then we should remove the `aria-hidden="true"` and any role. The alt text is useful here — a case may be made. Another case may be made that it is purely decorative, but then it would mean leaving out the alt text. I recommend keeping the alt text.

Reply:
I modded my comment and code snippet to match up to your comment @jenstrickland

Reply:
Thank you both for your input! I removed both the `role` and the `aria-hidden` properties.